### PR TITLE
add rules for - Automate case OCP-20800 for key-value secret

### PR DESCRIPTION
**Pull request by xiaocwan**
@yapei @yanpzhan @SSshahan Please review
Feature PR is https://github.com/openshift/cucushift/pull/6886

### CODE REVIEW

**Comment by akostadinov** [comment]
@yapei ?

File: lib/rules/web/admin_console/4.0/secrets.xyaml [hunk 1]
```diff
@@ -99,4 +133,7 @@ check_secret_type:
   element:
     selector:
       xpath: //dt[.='Type']/following::dd[1][.='<secret_type>']
-
+click_edit_secret:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.0/secrets.xyaml]: Seems it's not updated, no need to add `click_edit_secret` IMO, you can just use `click_one_dropdown_action` with `Edit Secret` set as `item`

Comment (xiaocwan) [reply lib/rules/web/admin_console/4.0/secrets.xyaml]: @yapei Updated and passed locally.

File: lib/rules/web/admin_console/4.0/secrets.xyaml [hunk 2]
```diff
@@ -32,12 +32,48 @@ create_ssh_key_source_secret:
...
       id: secret-name
     type: input
     op: send_keys <secret_name>
+set_pair_key:
+  element:
+    selector:
+      id: <key_index>-key
+    type: input
+    op: send_keys <key>
+set_pair_value:
```
[the remark this answers is not in the document]
Comment (yapei) [reply lib/rules/web/admin_console/4.0/secrets.xyaml]: Seems this is still missed, but it's ok to merge it now, remember to update later

**Comment by yapei** [comment]
LGTM

**Pull request merged**